en/docs/security-guidelines/secure-engineering-guidelines/dynamic-analysis-with-owasp-zap.md

```diff
@@ -16,199 +16,199 @@ This document provides details of all necessary steps for configuring the OWASP
 
-![Placeholder]({{#base_path#}}/assets/images/secure-coding-guidelines/zap-15.png){ .post-image }
+![Placeholder](../../assets/images/secure-coding-guidelines/zap-15.png)
 
 When searching also you can search URLs that belong only to the context with the **Search all URLs** option enabled. 
 
-![Placeholder]({{#base_path#}}/assets/images/secure-coding-guidelines/zap-16.png){ .post-image }
+![Placeholder](../../assets/images/secure-coding-guidelines/zap-16.png)
 
 You can also filter the alerts that belong only to the contexts you have added with the **Show all URLs** option enabled.
 
-![Placeholder]({{#base_path#}}/assets/images/secure-coding-guidelines/zap-17.png){ .post-image }
+![Placeholder](../../assets/images/secure-coding-guidelines/zap-17.png)
 
 
 ### Globally Excluding URL Patterns
-When the ZAP tool starts crawling the site, it will increase the network traffic heavily. We can reduce the traffic by excluding the URL patterns globally so that ZAP will ignore such URLs when crawling. For example if we exclude URLs of .mp4 video files, ZAP tool will not download mp4 video files which saves network bandwidth.
+When the ZAP tool starts crawling the site, it will increase the network traffic heavily. We can reduce the traffic by excluding the URL patterns globally so that ZAP will ignore such URLs when crawling. For example, if we exclude URLs of .mp4 video files, the ZAP tool will not download mp4 video files which saves network bandwidth.
 
-Go to **Tools** &rarr; **Options** and select Global Exclude URL option. By default there are some patterns already added to ZAP. You can select all of them for exclusion. Additionally, if there is any URL pattern you need to exclude, you can add the regex for the URL as a new entry. 
+Go to **Tools** &rarr; **Options** and select the Global Exclude URL option. By default, there are some patterns already added to ZAP. You can select all of them for exclusion. Additionally, if there is any URL pattern you need to exclude, you can add the regex for the URL as a new entry. 
 
-![Placeholder]({{#base_path#}}/assets/images/secure-coding-guidelines/zap-18.png){ .post-image }
+![Placeholder](../../assets/images/secure-coding-guidelines/zap-18.png)
 
 
 ### Creating the Logged in User Session
-When running the spider to crawl the site, we have to let ZAP discover the URLs that are accessible only by logged in users as well. For that, we need to create a logged in user session in ZAP, so that same as a logged in user can browse the URLs in a web browser, ZAP will be able to crawl through those URLs. 
+When running the spider to crawl the site, we have to let ZAP discover the URLs that are accessible only by logged-in users as well. For that, we need to create a logged-in user session in ZAP, so that same as a logged-in user can browse the URLs in a web browser, ZAP will be able to crawl through those URLs. 
```

Comment:
```suggestion
When running the spider to crawl the site, we have to let ZAP discover the URLs that are accessible only by logged-in users as well. For that, we need to create a logged-in user session in ZAP, so that in the same way that a logged-in user can browse the URLs in a web browser, ZAP will be able to crawl through those URLs. 
```